components/BlogPost/index.tsx

```diff
@@ -0,0 +1,33 @@
+import MarkdownRenderer from '../MarkdownRenderer';
+
+import styles from './style.module.css';
+
+
+import { CalendarOutlined } from  '@ant-design/icons';
+
+const BlogPost = (props: { data: { title: string; authorImage: string; author: string; date: string }; content: string; }) => {
```

Comment:
```suggestion
const BlogPost = ({ data: { title: string; authorImage: string; author: string; date: string }; content: string; }) => {
```

Use the fields directly in this component :muscle: